specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/preview/2019-05-01-preview/examples/UpdateImageTemplateTags.json

```diff
@@ -43,7 +43,10 @@
                 "tagName": "value"
               }
             }
-          ]
+          ],
+          "vmProfile": {
```

Comment:
Other than what's auto-generated by the autorest pipelines, no.
Just to understand what is a breaking change, are you referring to this kind (New property added to response)?
I think we've been doing something similar in the past, but probably just because we are still in preview. Few if any people use SDK or whatever is generated from the swagger. My understanding is that in preview, breaking changes are still allowed. However, we'll definitely be more aware of this kind of change when stable.